### ISSUE TITLE
Support twisted Edwards curves ?

### ISSUE THREAD

**Issue by ytrezq** [pwang00/RSA-Attacks#1]
Simple question, but it would be a nice addition since some cases aren’t handled by the conversion to the Wiestrass form.

**Comment by pwang00** [comment]
@ytrezq it's been a while since I've worked on this repo (or touched cryptography for that matter).  Are you talking about all the EC attacks?

**Comment by ytrezq** [comment]
I’m talking about all those where the code needs to be adapted to work on curves in the Twisted Edwards form : for example computing the order of a point in the Pohlig‑Hellman attack…

**Comment by ytrezq** [comment]
@pwang00 : Not to mention https://github.com/pwang00/Cryptographic-Attacks/blob/master/Public%20Key/Elliptic%20Curves/MOV_attack.sage seems rather empty.

**Comment by pwang00** [comment]
Yeah I've kind of been AFK from this repo for a while, busy with other stuff but will implement that when I can

To make sure I understand your request, you want the attacks to work directly on twisted curves without having to map back to Weierstrass, since some curves don't work?
